src/components/profile/index.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+// Components
+import UserProfileDisplay from './user';
+import DefaultProfileDisplay from './default';
+
+// MUI
+import withStyles from '@material-ui/core/styles/withStyles';
+import style from '../../style/style';
+
+// Redux
+import { connect } from 'react-redux';
+
+/** View component for displaying either the default or user profile
+ * @type {React.FunctionComponent}
+ * @param {object} props
+ * @param {object} props.classes
+ * @param {boolean} props.user.isLoading
+ * @param {boolean} props.user.authenticated
+ */
+const Profile = ({ classes, user: { isLoading, authenticated } }) => {
+  let profileMarkup;
+  if (!isLoading) {
+    if (authenticated) {
+      profileMarkup = <UserProfileDisplay classes={classes} />;
```

Comment:
you can return directly here rather than using the above let

Reply:
fixed